addons/web/static/src/views/pivot/pivot_model.js

```diff
@@ -1271,7 +1274,10 @@ export class PivotModel extends Model {
         let rows = [];
         const group = tree.root;
         const rowGroupId = [group.values, []];
-        const title = group.labels[group.labels.length - 1] || this.env._t("Total");
+        const title =
+            group.labels.length
+                ? group.labels[group.labels.length - 1]
+                : this.env._t("Total");
```

Comment:
```suggestion
                : _t("Total");
```
like above :-)

Reply:
`_t` is not available at this point. In above, there is `const _t = this.env._t` some lines above.